bin/waagent2.0

```diff
@@ -3457,9 +3457,21 @@ class RdmaHandler(object):
         """
         Write config string to /dev/hvnd_rdma
         """
-        Log("Set /dev/hvnd_rdma")
-        self.wait_rdma_dev()
-        self.write_rdma_dev_conf()
+        skip_rdma_device = False
```

Comment:
I'm not sure why this file was edited. We didn't do any rdma changes to this code for a long time. Now we expect all the RDMA "supported" distros to be on 2.1.x or above, so let's not edit this file.